packages/dialtone-tokens/package.json

```diff
@@ -23,18 +20,19 @@
   },
   "homepage": "https://dialtone.dialpad.com/",
   "devDependencies": {
-    "@tokens-studio/sd-transforms": "^0.14.2",
+    "@tokens-studio/sd-transforms": "^0.12.2",
```

Comment:
Per sd-transforms website we were actually using a version of sd-transforms that was incompatible with style-dictionary 3. We should really update to style-dictionary 4 but that's outside the scope of this PR.

Reply:
Does this version change in any way the format of the tokens? If so, it might impact the Figma variables sync process

Reply:
Yeah, wanted to ask just in case, but its a minor update so it should not change, thanks!